features/account/customer_account/viewing_orders_history/viewing_order_recalculated_by_exchange_rate.feature

```diff
@@ -5,17 +5,16 @@ Feature: Viewing details of an order
     I want to be able to view details of my placed order
```

Comment:
We have such a scenario in [checkout](https://github.com/Sylius/Sylius/blob/currency-party/features/checkout/placing_order_on_multi_channel/placing_an_order_on_a_single_channel_store.feature#L28)